### PR TITLE
[Geekeren] #348 Add axios instrumentation

### LINKED ISSUE #348: Add axios instrumentation
There are a number of broken axios instrumentation out there. Maybe it is time to have a first party one that has all the tests running etc? I'm not sure on this, so deferring to other maintainers to make the call.

@thegryfon has been working on fixups to https://github.com/geekeren/zipkin-js-instrumentation-axios 

@geekeren would you be interested in contributing yours?

### PR DESCRIPTION
**Pull request by geekeren**
Closes #348

### CODE REVIEW

**Comment by jcchavezs** [comment]
Ping @openzipkin/zipkin-js-champions

File: packages/zipkin-instrumentation-axios/package.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
+  "name": "zipkin-js-instrumentation-axios",
```
Comment (jcchavezs) [inline packages/zipkin-instrumentation-axios/package.json]: `zipkin-js-instrumentation-axios` -> `zipkin-instrumentation-axios`

File: packages/zipkin-instrumentation-axios/package.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  "description": "Library to instrument the axios HTTP-client to Support zipkin.",
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "mocha --compilers js:@babel/register --exit",
+    "build": "babel src --out-dir lib",
+    "prepublish": "babel src --out-dir lib"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/geekeren/zipkin-js-instrumentation-axios.git"
```
Comment (jcchavezs) [inline packages/zipkin-instrumentation-axios/package.json]: `s/geekeren/openzipkin`

File: packages/zipkin-instrumentation-axios/package.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  },
+  "keywords": [
+    "zipkin",
+    "open-trace",
```
Comment (jcchavezs) [inline packages/zipkin-instrumentation-axios/package.json]: I think this keyword `open-trace` is misleading as this repo does not support OT.

File: packages/zipkin-instrumentation-axios/package.json [hunk 4]
```diff
@@ -0,0 +1,53 @@
+{
+  "name": "zipkin-js-instrumentation-axios",
+  "version": "0.2.0",
```
Comment (jcchavezs) [inline packages/zipkin-instrumentation-axios/package.json]: Not sure we need to include this version here. Ping @ghermeto

File: packages/zipkin-instrumentation-axios/package.json (hunk 1 above)
Comment (geekeren) [reply packages/zipkin-instrumentation-axios/package.json]: The name ` zipkin-instrumentation-axios` was taken up, any suggestion?

Comment (jcchavezs) [reply packages/zipkin-instrumentation-axios/package.json]: Maybe `zipkin-instrumentation-axiosjs` or if we get @uschmann in the conversation maybe we can agree in a official one that could reuse the existing name?

**Comment by adriancole** [comment]
sounds ideal

**Comment by geekeren** [comment]
Could anyone or @adriancole help me to the reason why CI failed: https://travis-ci.org/openzipkin/zipkin-js/jobs/511854168 , I have spent some time to fix but failed.

**Comment by adriancole** [comment]
@geekeren maybe you can look at one of the other package.json files to see if stripping this down will avoid the error? like https://github.com/openzipkin/zipkin-js/blob/master/packages/zipkin-instrumentation-connect/package.json

The errors seems like typescript version mismatches and so removing more things here that aren't needed might be a good thing.

File: packages/zipkin-instrumentation-axios/package.json (hunk 1 above)
Comment (geekeren) [reply packages/zipkin-instrumentation-axios/package.json]: I already published this package with name: `zipkin-instrumentation-axiosjs` and openzipkin will be the maintainer

**Pull request merged**